critiquebrainz/frontend/forms/review.py

```diff
@@ -19,14 +18,41 @@ def __call__(self, form, field):
         if l < self.min or self.max != -1 and l > self.max:
             raise ValidationError(self.message)
 
+# Frequently Used Languages
+most_used_languages = []
+top_languages()
```

Comment:
This function call seems a little weird to me, can you explain what exactly you're trying to do here?